components/calendar/src/cal/mod.rs

```diff
@@ -39,6 +39,7 @@ pub use roc::Roc;
 pub use crate::any_calendar::{AnyCalendar, AnyCalendarKind};
 
 /// Internal scaffolding types
+#[cfg_attr(not(feature = "unstable"), doc(hidden))]
 pub mod scaffold {
```

Comment:
Thought: Types in `scaffold` are not unstable in the same way as types in `provider`; it is still safe to make bounds based on them. Discuss?

In this particular instance, `UnstableSealed` is good to tag in this way, but maybe not the whole module?

Reply:
Agreed.

 - UnstableSealed should be feature-gated
 - The traits themselves should be always-public: people will need them for bounds
 - The trait methods should be feature-hidden